[ray] Build failure when installing from source

<!--Please include [tune], [rllib], [autoscaler] etc. in the issue title if relevant-->

### What is the problem?
When trying to build ray from source I am getting the following output:

<details>
  <summary>Error</summary>
  
  ```
running build
running build_py
running egg_info
writing ray.egg-info/PKG-INFO
writing dependency_links to ray.egg-info/dependency_links.txt
writing entry points to ray.egg-info/entry_points.txt
writing requirements to ray.egg-info/requires.txt
writing top-level names to ray.egg-info/top_level.txt
reading manifest file 'ray.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
warning: no files found matching 'ray/python/ray/autoscaler/ray-schema.json'
writing manifest file 'ray.egg-info/SOURCES.txt'
running build_ext
+ set -e
+ SUPPORTED_PYTHONS=("3.5" "3.6" "3.7" "3.8")
+++ dirname ../build.sh
++ cd ..
++ pwd
+ ROOT_DIR=/home/acxz/vcs/git/github/acxz/pkgbuilds/python-ray-git/src/ray
++ uname
+ unamestr=Linux
+ [[ Linux == \L\i\n\u\x ]]
+ PARALLEL=1
+ RAY_BUILD_PYTHON=YES
+ RAY_BUILD_JAVA=NO
+ PYTHON_EXECUTABLE=
+ BUILD_DIR=
+ [[ 2 -gt 0 ]]
+ key=-p
+ case $key in
+ PYTHON_EXECUTABLE=/usr/bin/python
+ shift
+ shift
+ [[ 0 -gt 0 ]]
+ [[ -z /usr/bin/python ]]
++ /usr/bin/python -c 'import sys; version=sys.version_info[:3]; print("{0}.{1}".format(*version))'
+ PYTHON_VERSION=3.8
+ found=
+ for allowed in ${SUPPORTED_PYTHONS[@]}
+ [[ 3.8 == 3.5 ]]
+ for allowed in ${SUPPORTED_PYTHONS[@]}
+ [[ 3.8 == 3.6 ]]
+ for allowed in ${SUPPORTED_PYTHONS[@]}
+ [[ 3.8 == 3.7 ]]
+ for allowed in ${SUPPORTED_PYTHONS[@]}
+ [[ 3.8 == 3.8 ]]
+ found=3.8
+ break
+ [[ -z 3.8 ]]
+ echo 'Using Python executable /usr/bin/python.'
Using Python executable /usr/bin/python.
+ '[' -z '' ']'
++ PATH=/home/acxz/cf_ws/devel/bin:/opt/ros/noetic/bin:/home/acxz/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl:/home/acxz/.bazel/bin
++ which bazel
+ BAZEL_EXECUTABLE=/usr/bin/bazel
+ '[' -f /usr/bin/bazel ']'
+ echo 'Using Bazel executable /usr/bin/bazel.'
Using Bazel executable /usr/bin/bazel.
+ BUILD_DIR=/home/acxz/vcs/git/github/acxz/pkgbuilds/python-ray-git/src/ray/build/
+ '[' '!' -d /home/acxz/vcs/git/github/acxz/pkgbuilds/python-ray-git/src/ray/build/ ']'
+ pushd /home/acxz/vcs/git/github/acxz/pkgbuilds/python-ray-git/src/ray/build/
~/vcs/git/github/acxz/pkgbuilds/python-ray-git/src/ray/build ~/vcs/git/github/acxz/pkgbuilds/python-ray-git/src/ray/python
+ '[' NO == YES ']'
+ '[' YES == YES ']'
+ pickle5_available=0
+ pickle5_path=/home/acxz/vcs/git/github/acxz/pkgbuilds/python-ray-git/src/ray/python/ray/pickle5_files
+ check_pickle5_command='import sys\nif sys.version_info < (3, 8, 2): import pickle5;'
+ PYTHONPATH=/home/acxz/vcs/git/github/acxz/pkgbuilds/python-ray-git/src/ray/python/ray/pickle5_files:/home/acxz/cf_ws/devel/lib/python3.8/site-packages:/opt/ros/noetic/lib/python3.8/site-packages
+ /usr/bin/python -s -c 'exec("import sys\nif sys.version_info < (3, 8, 2): import pickle5;")'
+ pickle5_available=1
+ '[' 1 -ne 1 ']'
+ '[' -z '' ']'
+ CC=gcc
+ /usr/bin/python -m pip install -q psutil setproctitle --target=/home/acxz/vcs/git/github/acxz/pkgbuilds/python-ray-git/src/ray/python/ray/thirdparty_files
WARNING: Target directory /home/acxz/vcs/git/github/acxz/pkgbuilds/python-ray-git/src/ray/python/ray/thirdparty_files/setproctitle-1.1.10.dist-info already exists. Specify --upgrade to force replacement.
WARNING: Target directory /home/acxz/vcs/git/github/acxz/pkgbuilds/python-ray-git/src/ray/python/ray/thirdparty_files/setproctitle.cpython-38-x86_64-linux-gnu.so already exists. Specify --upgrade to force replacement.
WARNING: Target directory /home/acxz/vcs/git/github/acxz/pkgbuilds/python-ray-git/src/ray/python/ray/thirdparty_files/psutil already exists. Specify --upgrade to force replacement.
WARNING: Target directory /home/acxz/vcs/git/github/acxz/pkgbuilds/python-ray-git/src/ray/python/ray/thirdparty_files/psutil-5.7.0.dist-info already exists. Specify --upgrade to force replacement.
+ export PYTHON3_BIN_PATH=/usr/bin/python
+ PYTHON3_BIN_PATH=/usr/bin/python
+ /usr/bin/bazel build //:ray_pkg --verbose_failures
Loading:
Loading: 0 packages loaded
Analyzing: target //:ray_pkg (0 packages loaded, 0 targets configured)
INFO: Analyzed target //:ray_pkg (0 packages loaded, 0 targets configured).
INFO: Found 1 target...
[0 / 14] [Prepa] BazelWorkspaceStatusAction stable-status.txt
[1 / 14] Compiling src/ray/gcs/gcs_server/task_info_handler_impl.cc; 1s linux-sandbox ... (4 actions running)
[1 / 14] Compiling src/ray/gcs/gcs_server/task_info_handler_impl.cc; 11s linux-sandbox ... (4 actions running)
[2 / 14] Compiling src/ray/gcs/gcs_server/gcs_actor_manager.cc; 20s linux-sandbox ... (4 actions, 3 running)
[2 / 14] Compiling src/ray/gcs/gcs_server/gcs_actor_manager.cc; 25s linux-sandbox ... (4 actions running)
[3 / 14] Compiling src/ray/gcs/gcs_server/gcs_actor_manager.cc; 29s linux-sandbox ... (4 actions, 3 running)
[5 / 14] Compiling src/ray/gcs/gcs_server/gcs_server.cc; 36s linux-sandbox ... (4 actions, 3 running)
[6 / 14] Compiling src/ray/gcs/gcs_server/gcs_server.cc; 43s linux-sandbox ... (4 actions running)
[10 / 15] Compiling src/ray/gcs/gcs_server/worker_info_handler_impl.cc; 13s linux-sandbox ... (3 actions running)
[12 / 15] Executing genrule //:redis; 11s linux-sandbox ... (2 actions, 1 running)
[13 / 15] Executing genrule //:redis; 23s linux-sandbox
[13 / 15] Executing genrule //:redis; 47s linux-sandbox
ERROR: /home/acxz/vcs/git/github/acxz/pkgbuilds/python-ray-git/src/ray/BUILD.bazel:1551:8: Executing genrule //:redis failed (Exit 2): bash failed: error executing command
  (cd /home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray && \
  exec env - \
    LD_LIBRARY_PATH=/home/acxz/cf_ws/devel/lib:/opt/ros/noetic/lib:/usr/lib/gazebo-11/plugins:/usr/lib/gazebo-11/plugins \
    PATH=/home/acxz/cf_ws/devel/bin:/opt/ros/noetic/bin:/home/acxz/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl \
  /bin/bash -c 'source external/bazel_tools/tools/genrule/genrule-setup.sh;
            tmpdir="redis.tmp" &&
            path=external/com_github_antirez_redis/Makefile &&
            cp -p -L -R -- "${path%/*}" "${tmpdir}" &&
            chmod +x "${tmpdir}"/deps/jemalloc/configure &&
            parallel="$(getconf _NPROCESSORS_ONLN || echo 1)"
            make -s -C "${tmpdir}" -j"${parallel}" V=0 CFLAGS="${CFLAGS-} -DLUA_USE_MKSTEMP -Wno-pragmas -Wno-empty-body" &&
            mv "${tmpdir}"/src/redis-server bazel-out/k8-opt/bin/redis-server &&
            chmod +x bazel-out/k8-opt/bin/redis-server &&
            mv "${tmpdir}"/src/redis-cli bazel-out/k8-opt/bin/redis-cli &&
            chmod +x bazel-out/k8-opt/bin/redis-cli &&
            rm -r -f -- "${tmpdir}"
        ')
Execution platform: @local_config_platform//:host

Use --sandbox_debug to see verbose messages from the sandbox bash failed: error executing command
  (cd /home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray && \
  exec env - \
    LD_LIBRARY_PATH=/home/acxz/cf_ws/devel/lib:/opt/ros/noetic/lib:/usr/lib/gazebo-11/plugins:/usr/lib/gazebo-11/plugins \
    PATH=/home/acxz/cf_ws/devel/bin:/opt/ros/noetic/bin:/home/acxz/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl \
  /bin/bash -c 'source external/bazel_tools/tools/genrule/genrule-setup.sh;
            tmpdir="redis.tmp" &&
            path=external/com_github_antirez_redis/Makefile &&
            cp -p -L -R -- "${path%/*}" "${tmpdir}" &&
            chmod +x "${tmpdir}"/deps/jemalloc/configure &&
            parallel="$(getconf _NPROCESSORS_ONLN || echo 1)"
            make -s -C "${tmpdir}" -j"${parallel}" V=0 CFLAGS="${CFLAGS-} -DLUA_USE_MKSTEMP -Wno-pragmas -Wno-empty-body" &&
            mv "${tmpdir}"/src/redis-server bazel-out/k8-opt/bin/redis-server &&
            chmod +x bazel-out/k8-opt/bin/redis-server &&
            mv "${tmpdir}"/src/redis-cli bazel-out/k8-opt/bin/redis-cli &&
            chmod +x bazel-out/k8-opt/bin/redis-cli &&
            rm -r -f -- "${tmpdir}"
        ')
Execution platform: @local_config_platform//:host

Use --sandbox_debug to see verbose messages from the sandbox
src/malloc_io.c: In function 'malloc_vsnprintf':
src/malloc_io.c:369:2: warning: case label value exceeds maximum value for type [-Wswitch-outside-range]
  369 |  case '?' | 0x80:      \
      |  ^~~~
src/malloc_io.c:581:5: note: in expansion of macro 'GET_ARG_NUMERIC'
  581 |     GET_ARG_NUMERIC(val, 'p');
      |     ^~~~~~~~~~~~~~~
src/malloc_io.c:387:2: warning: case label value exceeds maximum value for type [-Wswitch-outside-range]
  387 |  case 'j' | 0x80:      \
      |  ^~~~
src/malloc_io.c:581:5: note: in expansion of macro 'GET_ARG_NUMERIC'
  581 |     GET_ARG_NUMERIC(val, 'p');
      |     ^~~~~~~~~~~~~~~
src/malloc_io.c:375:2: warning: case label value exceeds maximum value for type [-Wswitch-outside-range]
  375 |  case 'l' | 0x80:      \
      |  ^~~~
src/malloc_io.c:581:5: note: in expansion of macro 'GET_ARG_NUMERIC'
  581 |     GET_ARG_NUMERIC(val, 'p');
      |     ^~~~~~~~~~~~~~~
src/malloc_io.c:381:2: warning: case label value exceeds maximum value for type [-Wswitch-outside-range]
  381 |  case 'q' | 0x80:      \
      |  ^~~~
src/malloc_io.c:581:5: note: in expansion of macro 'GET_ARG_NUMERIC'
  581 |     GET_ARG_NUMERIC(val, 'p');
      |     ^~~~~~~~~~~~~~~
src/malloc_io.c:396:2: warning: case label value exceeds maximum value for type [-Wswitch-outside-range]
  396 |  case 'z' | 0x80:      \
      |  ^~~~
src/malloc_io.c:581:5: note: in expansion of macro 'GET_ARG_NUMERIC'
  581 |     GET_ARG_NUMERIC(val, 'p');
      |     ^~~~~~~~~~~~~~~
src/malloc_io.c: In function 'je_malloc_vsnprintf':
src/malloc_io.c:369:2: warning: case label value exceeds maximum value for type [-Wswitch-outside-range]
  369 |  case '?' | 0x80:      \
      |  ^~~~
src/malloc_io.c:581:5: note: in expansion of macro 'GET_ARG_NUMERIC'
  581 |     GET_ARG_NUMERIC(val, 'p');
      |     ^~~~~~~~~~~~~~~
src/malloc_io.c:387:2: warning: case label value exceeds maximum value for type [-Wswitch-outside-range]
  387 |  case 'j' | 0x80:      \
      |  ^~~~
src/malloc_io.c:581:5: note: in expansion of macro 'GET_ARG_NUMERIC'
  581 |     GET_ARG_NUMERIC(val, 'p');
      |     ^~~~~~~~~~~~~~~
src/malloc_io.c:375:2: warning: case label value exceeds maximum value for type [-Wswitch-outside-range]
  375 |  case 'l' | 0x80:      \
      |  ^~~~
src/malloc_io.c:581:5: note: in expansion of macro 'GET_ARG_NUMERIC'
  581 |     GET_ARG_NUMERIC(val, 'p');
      |     ^~~~~~~~~~~~~~~
src/malloc_io.c:381:2: warning: case label value exceeds maximum value for type [-Wswitch-outside-range]
  381 |  case 'q' | 0x80:      \
      |  ^~~~
src/malloc_io.c:581:5: note: in expansion of macro 'GET_ARG_NUMERIC'
  581 |     GET_ARG_NUMERIC(val, 'p');
      |     ^~~~~~~~~~~~~~~
src/malloc_io.c:396:2: warning: case label value exceeds maximum value for type [-Wswitch-outside-range]
  396 |  case 'z' | 0x80:      \
      |  ^~~~
src/malloc_io.c:581:5: note: in expansion of macro 'GET_ARG_NUMERIC'
  581 |     GET_ARG_NUMERIC(val, 'p');
      |     ^~~~~~~~~~~~~~~
redis-check-aof.c: In function 'consumeNewline.part.0':
redis-check-aof.c:37:20: warning: '%s' directive writing up to 1023 bytes into a region of size 1004 [-Wformat-overflow=]
   37 |     sprintf(error, "0x%16llx: %s", (long long)epos, __buf); \
      |                    ^~~~~~~~~~~~~~                   ~~~~~
redis-check-aof.c:45:9: note: in expansion of macro 'ERROR'
   45 |         ERROR("Expected \\r\\n, got: %02x%02x",buf[0],buf[1]);
      |         ^~~~~
redis-check-aof.c:37:31: note: format string is defined here
   37 |     sprintf(error, "0x%16llx: %s", (long long)epos, __buf); \
      |                               ^~
redis-check-aof.c:37:5: note: 'sprintf' output between 21 and 1044 bytes into a destination of size 1024
   37 |     sprintf(error, "0x%16llx: %s", (long long)epos, __buf); \
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
redis-check-aof.c:45:9: note: in expansion of macro 'ERROR'
   45 |         ERROR("Expected \\r\\n, got: %02x%02x",buf[0],buf[1]);
      |         ^~~~~
redis-check-aof.c: In function 'readLong':
redis-check-aof.c:37:20: warning: '%s' directive writing up to 1023 bytes into a region of size 1004 [-Wformat-overflow=]
   37 |     sprintf(error, "0x%16llx: %s", (long long)epos, __buf); \
      |                    ^~~~~~~~~~~~~~                   ~~~~~
redis-check-aof.c:45:9: note: in expansion of macro 'ERROR'
   45 |         ERROR("Expected \\r\\n, got: %02x%02x",buf[0],buf[1]);
      |         ^~~~~
redis-check-aof.c:37:31: note: format string is defined here
   37 |     sprintf(error, "0x%16llx: %s", (long long)epos, __buf); \
      |                               ^~
redis-check-aof.c:37:5: note: 'sprintf' output between 21 and 1044 bytes into a destination of size 1024
   37 |     sprintf(error, "0x%16llx: %s", (long long)epos, __buf); \
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
redis-check-aof.c:45:9: note: in expansion of macro 'ERROR'
   45 |         ERROR("Expected \\r\\n, got: %02x%02x",buf[0],buf[1]);
      |         ^~~~~
redis-check-aof.c:37:20: warning: '%s' directive writing up to 1023 bytes into a region of size 1004 [-Wformat-overflow=]
   37 |     sprintf(error, "0x%16llx: %s", (long long)epos, __buf); \
      |                    ^~~~~~~~~~~~~~                   ~~~~~
redis-check-aof.c:58:9: note: in expansion of macro 'ERROR'
   58 |         ERROR("Expected prefix '%c', got: '%c'",prefix,buf[0]);
      |         ^~~~~
redis-check-aof.c:37:31: note: format string is defined here
   37 |     sprintf(error, "0x%16llx: %s", (long long)epos, __buf); \
      |                               ^~
redis-check-aof.c:37:5: note: 'sprintf' output between 21 and 1044 bytes into a destination of size 1024
   37 |     sprintf(error, "0x%16llx: %s", (long long)epos, __buf); \
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
redis-check-aof.c:58:9: note: in expansion of macro 'ERROR'
   58 |         ERROR("Expected prefix '%c', got: '%c'",prefix,buf[0]);
      |         ^~~~~
redis-check-aof.c: In function 'readBytes':
redis-check-aof.c:37:20: warning: '%s' directive writing up to 1023 bytes into a region of size 1004 [-Wformat-overflow=]
   37 |     sprintf(error, "0x%16llx: %s", (long long)epos, __buf); \
      |                    ^~~~~~~~~~~~~~                   ~~~~~
redis-check-aof.c:70:9: note: in expansion of macro 'ERROR'
   70 |         ERROR("Expected to read %ld bytes, got %ld bytes",length,real);
      |         ^~~~~
redis-check-aof.c:37:31: note: format string is defined here
   37 |     sprintf(error, "0x%16llx: %s", (long long)epos, __buf); \
      |                               ^~
redis-check-aof.c:37:5: note: 'sprintf' output between 21 and 1044 bytes into a destination of size 1024
   37 |     sprintf(error, "0x%16llx: %s", (long long)epos, __buf); \
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
redis-check-aof.c:70:9: note: in expansion of macro 'ERROR'
   70 |         ERROR("Expected to read %ld bytes, got %ld bytes",length,real);
      |         ^~~~~
redis-check-aof.c: In function 'process':
redis-check-aof.c:37:20: warning: '%s' directive writing up to 1023 bytes into a region of size 1004 [-Wformat-overflow=]
   37 |     sprintf(error, "0x%16llx: %s", (long long)epos, __buf); \
      |                    ^~~~~~~~~~~~~~                   ~~~~~
redis-check-aof.c:120:25: note: in expansion of macro 'ERROR'
  120 |                         ERROR("Unexpected EXEC");
      |                         ^~~~~
redis-check-aof.c:37:31: note: format string is defined here
   37 |     sprintf(error, "0x%16llx: %s", (long long)epos, __buf); \
      |                               ^~
redis-check-aof.c:37:5: note: 'sprintf' output between 21 and 1044 bytes into a destination of size 1024
   37 |     sprintf(error, "0x%16llx: %s", (long long)epos, __buf); \
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
redis-check-aof.c:120:25: note: in expansion of macro 'ERROR'
  120 |                         ERROR("Unexpected EXEC");
      |                         ^~~~~
redis-check-aof.c:37:20: warning: '%s' directive writing up to 1023 bytes into a region of size 1004 [-Wformat-overflow=]
   37 |     sprintf(error, "0x%16llx: %s", (long long)epos, __buf); \
      |                    ^~~~~~~~~~~~~~                   ~~~~~
redis-check-aof.c:115:25: note: in expansion of macro 'ERROR'
  115 |                         ERROR("Unexpected MULTI");
      |                         ^~~~~
redis-check-aof.c:37:31: note: format string is defined here
   37 |     sprintf(error, "0x%16llx: %s", (long long)epos, __buf); \
      |                               ^~
redis-check-aof.c:37:5: note: 'sprintf' output between 21 and 1044 bytes into a destination of size 1024
   37 |     sprintf(error, "0x%16llx: %s", (long long)epos, __buf); \
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
redis-check-aof.c:115:25: note: in expansion of macro 'ERROR'
  115 |                         ERROR("Unexpected MULTI");
      |                         ^~~~~
redis-check-aof.c:37:20: warning: '%s' directive writing up to 1023 bytes into a region of size 1004 [-Wformat-overflow=]
   37 |     sprintf(error, "0x%16llx: %s", (long long)epos, __buf); \
      |                    ^~~~~~~~~~~~~~                   ~~~~~
redis-check-aof.c:136:9: note: in expansion of macro 'ERROR'
  136 |         ERROR("Reached EOF before reading EXEC for MULTI");
      |         ^~~~~
redis-check-aof.c:37:31: note: format string is defined here
   37 |     sprintf(error, "0x%16llx: %s", (long long)epos, __buf); \
      |                               ^~
redis-check-aof.c:37:5: note: 'sprintf' output between 21 and 1044 bytes into a destination of size 1024
   37 |     sprintf(error, "0x%16llx: %s", (long long)epos, __buf); \
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
redis-check-aof.c:136:9: note: in expansion of macro 'ERROR'
  136 |         ERROR("Reached EOF before reading EXEC for MULTI");
      |         ^~~~~
/usr/bin/ld: server.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: sds.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: ziplist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: networking.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: util.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: object.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: db.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: replication.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: rdb.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: t_string.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: t_list.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: t_set.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: t_zset.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: t_hash.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: config.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: aof.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: pubsub.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: multi.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: debug.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: sort.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: syncio.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: cluster.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: crc16.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: slowlog.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: scripting.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: bio.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: rio.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: bitops.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: sentinel.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: notify.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: blocked.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: hyperloglog.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: latency.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: sparkline.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: redis-check-rdb.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: redis-check-aof.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: geo.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: lazyfree.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: module.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: evict.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: expire.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: childinfo.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: defrag.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: t_stream.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: lolwut.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
/usr/bin/ld: lolwut5.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: multiple definition of `SDS_NOINIT'; quicklist.o:/home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray/redis.tmp/src/sds.h:37: first defined here
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:219: redis-server] Error 1
make[1]: *** Waiting for unfinished jobs....
make: *** [Makefile:6: all] Error 2
Target //:ray_pkg failed to build
ERROR: /home/acxz/vcs/git/github/acxz/pkgbuilds/python-ray-git/src/ray/BUILD.bazel:1629:8 Executing genrule //:redis failed (Exit 2): bash failed: error executing command
  (cd /home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray && \
  exec env - \
    LD_LIBRARY_PATH=/home/acxz/cf_ws/devel/lib:/opt/ros/noetic/lib:/usr/lib/gazebo-11/plugins:/usr/lib/gazebo-11/plugins \
    PATH=/home/acxz/cf_ws/devel/bin:/opt/ros/noetic/bin:/home/acxz/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl \
  /bin/bash -c 'source external/bazel_tools/tools/genrule/genrule-setup.sh;
            tmpdir="redis.tmp" &&
            path=external/com_github_antirez_redis/Makefile &&
            cp -p -L -R -- "${path%/*}" "${tmpdir}" &&
            chmod +x "${tmpdir}"/deps/jemalloc/configure &&
            parallel="$(getconf _NPROCESSORS_ONLN || echo 1)"
            make -s -C "${tmpdir}" -j"${parallel}" V=0 CFLAGS="${CFLAGS-} -DLUA_USE_MKSTEMP -Wno-pragmas -Wno-empty-body" &&
            mv "${tmpdir}"/src/redis-server bazel-out/k8-opt/bin/redis-server &&
            chmod +x bazel-out/k8-opt/bin/redis-server &&
            mv "${tmpdir}"/src/redis-cli bazel-out/k8-opt/bin/redis-cli &&
            chmod +x bazel-out/k8-opt/bin/redis-cli &&
            rm -r -f -- "${tmpdir}"
        ')
Execution platform: @local_config_platform//:host

Use --sandbox_debug to see verbose messages from the sandbox bash failed: error executing command
  (cd /home/acxz/.cache/bazel/_bazel_acxz/890bc6e5a776c47ccca8b92872d9194a/sandbox/linux-sandbox/131/execroot/com_github_ray_project_ray && \
  exec env - \
    LD_LIBRARY_PATH=/home/acxz/cf_ws/devel/lib:/opt/ros/noetic/lib:/usr/lib/gazebo-11/plugins:/usr/lib/gazebo-11/plugins \
    PATH=/home/acxz/cf_ws/devel/bin:/opt/ros/noetic/bin:/home/acxz/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl \
  /bin/bash -c 'source external/bazel_tools/tools/genrule/genrule-setup.sh;
            tmpdir="redis.tmp" &&
            path=external/com_github_antirez_redis/Makefile &&
            cp -p -L -R -- "${path%/*}" "${tmpdir}" &&
            chmod +x "${tmpdir}"/deps/jemalloc/configure &&
            parallel="$(getconf _NPROCESSORS_ONLN || echo 1)"
            make -s -C "${tmpdir}" -j"${parallel}" V=0 CFLAGS="${CFLAGS-} -DLUA_USE_MKSTEMP -Wno-pragmas -Wno-empty-body" &&
            mv "${tmpdir}"/src/redis-server bazel-out/k8-opt/bin/redis-server &&
            chmod +x bazel-out/k8-opt/bin/redis-server &&
            mv "${tmpdir}"/src/redis-cli bazel-out/k8-opt/bin/redis-cli &&
            chmod +x bazel-out/k8-opt/bin/redis-cli &&
            rm -r -f -- "${tmpdir}"
        ')
Execution platform: @local_config_platform//:host

Use --sandbox_debug to see verbose messages from the sandbox
INFO: Elapsed time: 97.028s, Critical Path: 50.48s
INFO: 10 processes: 10 linux-sandbox.
FAILED: Build did NOT complete successfully
FAILED: Build did NOT complete successfully
Traceback (most recent call last):
  File "setup.py", line 205, in <module>
    setup(
  File "/usr/lib/python3.8/site-packages/setuptools/__init__.py", line 161, in setup
    return distutils.core.setup(**attrs)
  File "/usr/lib/python3.8/distutils/core.py", line 148, in setup
    dist.run_commands()
  File "/usr/lib/python3.8/distutils/dist.py", line 966, in run_commands
    self.run_command(cmd)
  File "/usr/lib/python3.8/distutils/dist.py", line 985, in run_command
    cmd_obj.run()
  File "/usr/lib/python3.8/distutils/command/build.py", line 135, in run
    self.run_command(cmd_name)
  File "/usr/lib/python3.8/distutils/cmd.py", line 313, in run_command
    self.distribution.run_command(command)
  File "/usr/lib/python3.8/distutils/dist.py", line 985, in run_command
    cmd_obj.run()
  File "setup.py", line 121, in run
    subprocess.check_call(command)
  File "/usr/lib/python3.8/subprocess.py", line 364, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['../build.sh', '-p', '/usr/bin/python']' returned non-zero exit status 1.

  ```
</details>


*Ray version and other system information (Python version, TensorFlow version, OS):*
ray: master
python: 3.8.3
gcc: 10.1.0
bazel: 3.2.0
os: Arch Linux

### Reproduction (REQUIRED)
Please provide a script that can be run to reproduce the issue. The script should have **no external library dependencies** (i.e., use fake or mock data / environments):

If we cannot run your script, we cannot fix your issue.

- [ ] I have verified my script runs in a clean environment and reproduces the issue.
- [ ] I have verified the issue also occurs with the [latest wheels](https://docs.ray.io/en/latest/installation.html).

The above does not apply since this is a build issue.

--- a/bazel/ray_deps_setup.bzl
+++ b/bazel/ray_deps_setup.bzl
@@ -75,8 +75,8 @@ def ray_deps_setup():
     auto_http_archive(
         name = "com_github_antirez_redis",
         build_file = "//bazel:BUILD.redis",
-        url = "https://github.com/antirez/redis/archive/5.0.3.tar.gz",
-        sha256 = "7084e8bd9e5dedf2dbb2a1e1d862d0c46e66cc0872654bdc677f4470d28d84c5",
+        url = "https://github.com/antirez/redis/archive/5.0.9.tar.gz",
+        sha256 = "db9bf149e237126f9bb5f40fb72f33701819555d06f16e9a38b4949794214201",
         patches = [
             "//thirdparty/patches:hiredis-connect-rename.patch",
             "//thirdparty/patches:hiredis-windows-sigpipe.patch",

